wayshot/src/cli.rs

```diff
@@ -1,43 +1,46 @@
-use std::path::PathBuf;
-
-use clap::arg;
-
-use clap::Parser;
-use eyre::WrapErr;
-
 use crate::utils::EncodingFormat;
-use clap::builder::TypedValueParser;
+use clap::Parser;
+use std::path::PathBuf;
+use tracing::Level;
 
 #[derive(Parser)]
 #[command(version, about)]
 pub struct Cli {
-    /// Custom output path can be of the following types:
-    ///     1. Directory (Default naming scheme is used for the image output).
+    /// Custom screenshot file path can be of the following types:
+    ///     1. Directory (Default naming scheme is used for the image screenshot file).
     ///     2. Path (Encoding is automatically inferred from the extension).
-    ///     3. `-` (Indicates writing to terminal [stdout]).
-    #[arg(value_name = "OUTPUT", verbatim_doc_comment)]
+    ///     3. None (the screenshot file with default filename_format will be created in current directory)
+    #[arg(value_name = "FILE", verbatim_doc_comment)]
     pub file: Option<PathBuf>,
 
-    /// Copy image to clipboard. Can be used simultaneously with [OUTPUT] or stdout.
+    /// Write screenshot to terminal/stdout.
+    /// Defaults to config value (`false`)
+    #[arg(long, verbatim_doc_comment)]
+    pub stdout: Option<bool>,
```

Comment:
See my comment in #100. Additionally, let's not change the CLI in this PR because this is already a large change. If you want to change the CLI, please do this in a followup PR and allow for a separate discussion and keep this reviewable.

So I would prefer to see this reverted and have this discussed in a future PR.

Reply:
This makes sense, lets do CLI changes separately.